configure.py

```diff
@@ -67,6 +67,13 @@ def parse_arguments():
       default=False,
       action=argparse.BooleanOptionalAction,
   )
+  parser.add_argument(
+      "--cuda-runner",
```

Comment:
What happens when I run `configure.py` without this argument and then run all tests? (run_tests.py)

Reply:
Good point, I talked to Nicolas about that and I do need to change the script to only enable cuda tests on demand